docs/tutorials/building-your-first-bot.mdx

```diff
@@ -0,0 +1,342 @@
+        }
+        
+        [RGAction("MoveInDirection")]
+        public void MoveInput(Vector2 newMoveDirection)
+        {
+            move = newMoveDirection;
+        }
+        
+        public void JumpInput(bool newJumpState)
+        {
+            jump = newJumpState;
+        }
+    }
+}
+```
+
+Then, open the `Assets/FirstBotDemo/Demo/Scripts/Player/PlayerAttack.cs` file again, and add the following method to the class,
+which defines an action that allows a bot to target an enemy and use an ability:
+
+```csharp
+[RGAction]
+public void SelectAndAttackEnemy(int enemyId, int ability)
+{
+    var enemy = RGFindUtils.Instance.FindOneByInstanceId<EnemyController>(enemyId);
+    SelectEnemy(enemy);
+    Attack(abilities[ability]);
+}
+```
+
+
+As a last step here, in the player input code, you will see that we introduced an `isBot` field to the `PlayerInputControl` component.
+This will allow our player to be controlled by the bot logic when it is set to true. However, there is an existing `Player`
+GameObject in the scene, which we want to control using our manual input controls. Click this `Player` in the object hierarchy,
+find the `Player Input Control` component, set `Is Bot` to unchecked, and then save (you may find that it is already unchecked).
+
+![Screenshot of the player prefab with the components added.](./img/building-your-first-bot/is_bot.png)
+
+That's it for actions! A bot may now call into two actions (moving and attacking), which we will demonstrate in the section
+when we build our bot.
+
+## Using a ready-to-go bot from the Bot Marketplace
+
+Now that our integration is complete, our final step is to use an implementation of a bot. We will start with a template
+found in our [Bot Marketplace](https://play.regression.gg/marketplace), and then will show you where to edit its code.
+
+Go to [this page](https://play.regression.gg/marketplace/1) to see the existing bot, and click "Create Bot" to create a copy 
+of the bot. On the bot creation page, give it a name such as "KitingBot", and then click "Create Bot". You will then be
+taken to the Agent Builder tool; feel free to click around and see how the bot works!
+
+![Screenshot of the bot creation page.](./img/building-your-first-bot/marketplace_bot.png)
+
+![Screenshot of the bot creation page.](./img/building-your-first-bot/create_bot_9.png)
+
+![Screenshot of the agent builder configuration for this bot.](./img/building-your-first-bot/agent_builder.png)
+
+Go back to Unity, and within the **Regression Games** menu in the menu bar, click **Synchronize Bots with RG**. This will pull your new
+bot into your Unity project. 
+
+![Screenshot of the agent builder configuration for this bot.](./img/building-your-first-bot/synchronize_bots.png)
+
+At this point, you can click play, and then select the Regression Games overlay button in the bottom
+right of your game screen. Use the dropdown to select the bot you just created, and then click "Start Bot". The bot will begin to
```

Comment:
```suggestion
right corner of your Scene. Use the dropdown to select the bot you just created, and then click "Start Bot". The bot will begin to
```

Reply:
Same comment as above regarding scene vs screen - I do mean the screen here. Unless viewport is preferred? I suppose according to unity, "Scene View" is the most correct term, so I can change it to that.

Reply:
Fixed